cleaning_data_review.py

```diff
@@ -0,0 +1,156 @@
+#Changes documents to UTF-8
+#Deletes Gutenberg part
+import os
+import glob
+import re
+
+def read_text_files_from_subfolders(main_folder_path):
+# Find all .txt files in the specified main folder and its subfolders

```

Comment:
please use docstring `"""comment"""` instead of one-line comments `# something` tl describe purpose of function